checkpoint/cp-resource-management/templates/serviceaccount.yaml

```diff
@@ -1,10 +1,10 @@
-{{ if .Values.serviceAccount.create }}
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: {{ template "cp-resource-management.serviceAccountName" . }}
-  labels:
-    app.kubernetes.io/name: {{ template "cp-resource-management.name" . }}
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-{{ end }}
+{{ if .Values.serviceAccount.create }}
```

Comment:
you are using the same service account RBAC and PSP. May it happe you will have .Values.serviceAccount.create false and .Values.rbac.pspEnable true? If so - change this "if" to be "create OR pspEnable"